task04/task04/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task04
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int x;
+            Console.WriteLine("Введите значение x");
+            x = int.Parse(Console.ReadLine());
```

Comment:
Неверный тип данных.

> Составить программу, которая запрашивает у пользователя **действительное** значение _x_...

```suggestion
            douible x;
            Console.WriteLine("Введите значение x");
            x = double.Parse(Console.ReadLine());
```